src/components/StatusUpdater/index.tsx

```diff
@@ -7,23 +7,54 @@ import { UserInfo } from '@respond/types/userInfo';
 import { SplitButton } from '../SplitButton';
 import { useFormLogic, UpdateStatusForm } from './UpdateStatusForm';
 
-const options = [
-  { id: ResponderStatus.Unavailable, text: 'Not Available' },
-  { id: ResponderStatus.Standby, text: 'Stand By' },
-  { id: ResponderStatus.SignedIn, text: 'Sign In' },
-  { id: ResponderStatus.SignedOut, text: 'Sign Out' },
-]
-const optionTexts = options.reduce((accum, cur) => ({ ...accum, [cur.id]: cur.text }), {} as Record<string, string>);
+const statusTransitions = {
+  standBy: { id: 0, newStatus: ResponderStatus.Standby, text: 'Stand By' },
+  standDown: { id: 1, newStatus: ResponderStatus.SignedOut, text: 'Stand Down' },
+  inTown: { id: 2, newStatus: ResponderStatus.Remote, text: 'In Town' },
+  signIn: { id: 3, newStatus: ResponderStatus.SignedIn, text: 'Sign In' },
+  turnAround: { id: 4, newStatus: ResponderStatus.Demobilized, text: 'Turn Around' },
+  arriveBase: { id: 5, newStatus: ResponderStatus.Available, text: 'Arrive Base' },
+  departBase: { id: 6, newStatus: ResponderStatus.Demobilized, text: 'Depart Base' },
+  signOut: { id: 7, newStatus: ResponderStatus.SignedOut, text: 'Sign Out' },
+  resetStatus: { id: 8, newStatus: ResponderStatus.NotResponding, text: 'Reset Status' }, // clear status in edge cases that shouldn't generally be possible.

```

Comment:
I don't know where it would do it yet, but it would be good to at least log if we ever hit this. Don't think it's worth presenting an error to the user, but for debugging purposes it would be nice to have something in the logs.

Reply:
Do we have somewhere to send logs?

Reply:
For now just `console.log(...)`.

Reply:
Yeah, I mean... if your just concerned about seeing it in real time... this is the only use case where "Not Responding" would be the button option. I wouldn't be logging anything beyond that. Seems redundant.